Dockerfile

```diff
@@ -1,6 +1,6 @@
 # PostgreSQL service with pghashlib and PL-Proxy
 
-FROM postgres:12
+FROM postgres:10.17
```

Comment:
This is a downgrade from 12 to 10, not an upgrade from 9.6 as the PR description notes. Can you explain that so we have a record of why it was using v12 (instead of 9.6) and the reason for this change?

Reply:
Sorry, Daniel. Thanks for correcting.
Sure, I will find the why 12 is mentioned in the ticket.

Reply:
Got to know from Simon that..

> this was done when investigating the possibility of upgrading PG to v12 in 2018. The 'latest' tag on dockerhub is still 9.6 which is what is used by the commcare-hq repo.

and Simon is fine with Merging this PR